httpapi/httpapi.go

```diff
@@ -62,7 +62,7 @@ func (h HTTPAPI) getFunction(w http.ResponseWriter, r *http.Request, params http
 		encoder.Encode(fn)
 	}
 
-	metricFunctionGetRequests.WithLabelValues(space).Inc()
+	metricConfigRequests.WithLabelValues(space, "function", "get").Inc()
 }
 
 func (h HTTPAPI) getFunctions(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
```

Comment:
Small nit, but is it worth making these handler names line up with the labels put in the metrics? `listFunctions`, `createFunction`, etc.

Reply:
True story.